client/README.md

```diff
@@ -1,3 +1,20 @@
-# React-Stuff
+# Book list Client:
 
-> Please fix my bugs!
\ No newline at end of file
+To start server initially:
+
+- Run the commands:
+
+```js
+npm install
+npm start
+```
+
+- After initial setup:
+
+```js
+npm start
+```
+
+The client can be accessed at the http://localhost:3000/ address in your browser by default.
+
+A react application to log and store your books, as well as the rating you give that book. Too, whether it was read or not.
```

Comment:
To start server initially:

- Run the following commands in your terminal while inside the `client` folder:

```
$ npm install
$ npm start
```

- After initial setup:

```js
$ npm start
```

The client can be accessed at the http://localhost:3000/ address in your browser by default.

A react application to log and store your books, as well as the rating you give that book. Too, whether it was read or not.

Reply:
Please get rid of the `js` portion of your three back-tick code blocks and put single back-tick around the word `client`.

Please add the `$` in front of each command that will be run in the terminal.